### ISSUE TITLE
[Warning] Usage of a deprecated API in the debug console

### ISSUE THREAD

**Issue by CodingLegend27** [tanersener/flutter-ffmpeg#246]
**Description**
When I use flutter-ffmpeg and I run my application on a device, I have the following warning: 
``` 
Note: /users/chrissi/development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_ffmpeg-0.3.0/android/src/main/java/com/arthenica/flutter/ffmpeg/FlutterFFmpegPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```
**Current behavior**
The app works fine but there's the warning in the debug console.

**Environment**
`flutter_ffmpeg: ^0.3.0`
`flutter version: 1.25.0-8.1.pre`

_Flutter doctor:_
```
[✓] Flutter (Channel beta, 1.25.0-8.1.pre, on macOS 11.1 20C69 darwin-x64, locale de-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2020.3)
[✓] VS Code (version 1.52.1)
[✓] Connected device (2 available)
```

**Comment by tanersener** [comment]
Thanks for reporting this. I plan to resolve those depreciation warnings in the next release.

**Comment by tanersener** [comment]
Depreciation warnings are resolved in `v0.4.0`.
